Kruskal_Algorithm/Kruskal_Algorithm.c

```diff
@@ -0,0 +1,137 @@
+// Kruskal Algorithm in C
+#include <stdio.h>
+#define NUM 20
+
+// Creating the edge
+typedef struct edge {
+	int p;          // Variable to contain all the first node of the edge
+	int q;          // Variable to contain all the second node of the edge  
+	int weight;     // Variable to contain the weight of the edge
+} edge;
+ 
+// Creating the Edgelist
+typedef struct edgelist {
+    edge data[NUM];
+	int n;
+} edgelist;
+ 
+edgelist list;
+int weights[NUM][NUM];          // Array for weights in the graph
+int n;
+edgelist listSpan;
+ 
+void kruskal_algo();
+void print();
+int find(int contain[], int vertexno);
+void union1(int contain[], int c1, int c2);
+void sort();
+
+
+// Driver Program
+int main() {
+	int total_cost; 
```

Comment:
Please use 4 space indentation

Reply:
@somya-kapoor I am not knowing about 4 space indentation. Can you please tell me one time by an example.